### PR TITLE
Feature/scastie front page

### PR DESCRIPTION
**Pull request by MasseGuillaume**
fix https://github.com/scala/scala-lang/pull/697

### CODE REVIEW

File: resources/js/functions.js [hunk 1]
```diff
@@ -234,33 +234,52 @@ $(document).ready(function() {
...
+
+    editor.setSize("100%", ($("#scastie-code-container").height()));
+
+    var codeSnippet = "List(\"Hello\", \"World\").mkString(\"\", \", \", \"!\")";
+    editor.getDoc().setValue(codeSnippet);
+    editor.refresh();
+
+    function run(){
+      console.log("run");
+      // var scastieBaseUrl = "https://scastie.scala-lang.org";
```
Comment (MasseGuillaume) [inline resources/js/functions.js]: I'm redeploying scastie, I will force push with the url pointing on the prod server.

Comment (heathermiller) [inline resources/js/functions.js]: Ok, cool. Let me know when it's up, we can test to make sure it's all good, and then we fix this.

**Comment by MasseGuillaume** [comment]
@heathermiller This PR is good to go.
out

**Comment by heathermiller** [comment]
❤️

File: _includes/scastie.html [hunk 2]
```diff
@@ -20,4 +26,4 @@
 			</div>
 		</div>
 	</div>
-</section>
\ No newline at end of file
+</section>m
```
Comment (heathermiller) [inline _includes/scastie.html]: Broken

**Comment by heathermiller** [comment]
All good now

**Pull request merged**